specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-08-02-preview/managedClusters.json

```diff
@@ -3556,9 +3556,90 @@
         "clientId"
       ]
     },
+    "FlexProfile": {
+      "type": "object",
+      "properties": {
+        "scaleProfile": {
+          "$ref": "#/definitions/ScaleProfile",
+          "description": "Specifications on how to scale a Flex agent pool."
+        }
+      },
+      "description": "Specifications on Flex agent pool."
+    },
+    "ScaleProfile": {
+      "type": "object",
+      "properties": {
+        "manual": {
+          "$ref": "#/definitions/ManualScaleProfile",
+          "description": "Specifications on how to scale the Flex agent pool to a fixed size."
+        }
+      },
+      "description": "Specifications on how to scale a Flex agent pool."
+    },
+    "ManualScaleProfile": {
+      "type": "object",
+      "properties": {
+        "machines": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManualScaleMachines"
+          },
+          "description": "Specifications on how to scale a Flex agent pool."
```

Comment:
If only a single value is allowed here now, consider mentioning that in the description?

Reply:
Added